packages/graph/migrations/postgres/migration/1656417312397_initial.ts

```diff
@@ -328,27 +328,22 @@ export async function up(pgm: MigrationBuilder): Promise<void> {
         notNull: true,
         references: "link_types",
       },
-      multi: {
-        type: "boolean",
-        notNull: true,
-        default: "false",
-      },
-      multi_order: {
+      link_order: {
         // TODO: this is where we could do fractional indexing
         //  https://app.asana.com/0/1200211978612931/1202085856561975/f
         type: "integer",
+        notNull: false,
       },
       created_by: {
         type: "UUID",
         notNull: true,
         references: "accounts",
       },
-      // TODO: Consider using timestamps for link duration like we've done for the HASH backend,
-      //   see https://app.asana.com/0/1201095311341924/1201836485518642/f
-      active: {
-        type: "boolean",
+      created_at: {
+        type: "TIMESTAMP WITH TIME ZONE",
         notNull: true,
-        default: true,
+        // This default would give the UTC time "wall clock" even within a transaction/
+        default: pgm.func("clock_timestamp()"),
```

Comment:
As we are explicit when inserting, it's probably better to not set a default value.

Reply:
Addressed in 708836711944a5a6b4f6c71215adbbb77b22b58d, had to add explicit `clock_timestamp` to `created_by` in 7e6e1d2908e5fad4556c028d63aab737a7cc7982